pkgs/desktops/gnome-3/extensions/gtile/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, glib }:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-gtile";
+  version = "34";
+
+  src = fetchFromGitHub {
+    owner = "gTile";
+    repo = "gTile";
+    rev = "V${version}";
+    sha256 = "1yka4acmrykh4vdd89yvn6yv89c6q16r72lgi3j48my3jghwjiwp";
+  };
+
+  uuid = "gTile@vibou";
+
+  nativeBuildInputs = [ glib ];
+
+  installPhase = ''
```

Comment:
When overriding `installPhase`, you should also provide `runHook preInstall` and `runHook postInstall`. Though, in this case, it is better to rely on the default `installPhase`.

Reply:
I just used the same approach which is used is another extensions within gnome-3 extensions directory. In general, i think that nix files in that directory should be refactored, to look consistent.